projects/client/src/lib/sections/summary/components/comments/Comments.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  import SectionList from "$lib/components/lists/section-list/SectionList.svelte";
+  import * as m from "$lib/features/i18n/messages.ts";
+  import type { MediaEntry } from "$lib/requests/models/MediaEntry";
+  import CommentCard from "./_internal/CommentCard.svelte";
+  import { useComments } from "./_internal/useComments";
+
+  const { media }: { media: MediaEntry } = $props();
+
+  const { isLoading, comments } = useComments({
+    slug: media.slug,
+    type: media.type,
+  });
```

Comment:
`$derived(useComments(...))`

Else navigating from one item to the next will render same data 😛